pkg/controller/operators/catalog/operator.go

```diff
@@ -69,6 +67,13 @@ func NewOperator(kubeconfigPath string, wakeupInterval time.Duration, operatorNa
 		subSharedIndexInformers = append(subSharedIndexInformers, nsInformerFactory.Subscription().V1alpha1().Subscriptions().Informer())
 	}
 
+	// Create an informer for each catalog namespace
+	catsrcSharedIndexInformers := []cache.SharedIndexInformer{}
+	for _, namespace := range []string{operatorNamespace} {
```

Comment:
Why `range` over a slice with a single item?

Reply:
Just because I know we'll implement "local" catalogs in the near future, where we'll want to iterate over `operatorNamespace ∪ watchedNamespaces` instead of just `operatorNamespace`